apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -177,7 +178,13 @@ export default function ListBox({
     model,
   });
 
-  const sizes = getListSizes({ layout, width, height, listCount: unlimitedListCount, count, textWidth });
+  let freqIsAllowed = getFrequencyAllowed({ itemWidth: width, layout, frequencyMode });
+  const sizes = getListSizes({ layout, width, height, listCount: unlimitedListCount, count, textWidth, freqIsAllowed });
+  if (sizes.columnWidth) {
```

Comment:
can this be defined when not in grid mode? Maybe return and use `isGridMode` from `getListSizes`?

Reply:
Yeah, I checked that up. `columnWidth` is only defined in grid mode, i.e. it's an indicator of grid mode, so it should be ok.